demos/msp432p401r/system_controller/source/main.cpp

```diff
@@ -2,6 +2,8 @@
 
 #include "L0_Platform/msp432p401r/msp432p401r.h"
 #include "L1_Peripheral/cortex/system_timer.hpp"
+#include "L1_Peripheral/msp432p401r/gpio.hpp"
+#include "L1_Peripheral/msp432p401r/pin.hpp"
 #include "L1_Peripheral/msp432p401r/system_controller.hpp"
 #include "utility/bit.hpp"
```

Comment:
It looks like you are not using this include anymore.

Reply:
Pin is used to configure the clock out pins and GPIO is used for the LED.